msal4j-brokers/pom.xml

```diff
@@ -3,22 +3,135 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <packaging>jar</packaging>
     <groupId>com.microsoft.azure</groupId>
     <artifactId>msal4j-brokers</artifactId>
     <version>0.0.1</version>
+    <packaging>jar</packaging>
+    <name>msal4j-brokers</name>
+    <description>
+        Microsoft Authentication Library for Java - Brokers helps you integrate with the broker
+        on windows machine to secure Access tokens and refresh tokens.
+    </description>
+    <url>https://github.com/AzureAD/microsoft-authentication-library-for-java</url>
+    <developers>
+        <developer>
+            <id>msopentech</id>
+            <name>Microsoft Open Technologies, Inc.</name>
```

Comment:
It was mentioned in a comment on your PR for the build script in the interop layer that 'Microsoft Open Technologies, Inc.' seems like a weird thing to put here, and honestly I'm not sure why this is even in our pom.xml for MSAL Java.

The <developer> tag is meant to be an optional documentation thing to just describe the developers, and I think we can just remove it from our pom's since it's not really necessary: https://maven.apache.org/pom.html#Developers

Reply:
Good point. It is interesting that some of the tags are optional but were giving me errors if not included. I dont remember if developer tag was one of them but to make it work, I just included everything from msal java pom xml. Lets talk and research more on why it was included in the first place before removing them. If they can be removed, we will remove them from both the modules.